docs/how-to/upgrade-cloud.rst

```diff
@@ -2,6 +2,104 @@
 Upgrade a cloud
 ===============
 
+Recommendations for upgrading production clouds
+-----------------------------------------------
+
+1. Be sure to upgrade in a maintenance window
+2. Start with the upgrade on control-plane applications
+3. Don't force the upgrade on non-empty hypervisors
+4. After upgrading the **control-plane**, choose one empty hypervisor to be a `canary node`.
+    After upgrading it, test if it's behaving as expected.
```

Comment:
Maybe due to the extra indention on the second line here, this item has a weird format after rendering. Could you take a look at it?